server/models/services/helpers.js

```diff
@@ -0,0 +1,42 @@
+const {
+    At,
+    AtMode,
+    AtVersion,
+    Browser,
+    BrowserVersion,
+    Role,
+    TestPlan,
+    TestPlanReport,
+    TestPlanRun,
+    TestPlanTarget,
+    TestResult,
+    User,
+    UserRoles
+} = require('../index');
+
+/**
+ * Use to retrieve all the attributes defined for a Sequelize model
+ * @param {Model} model - Sequelize model
+ * @returns {string[]}
+ */
+const getSequelizeModelAttributes = model => {
+    if (!model) throw new Error('Model not defined');
+
+    return Object.keys(model.rawAttributes).map(key => key);
+};
+
+module.exports = {
+    AT_ATTRIBUTES: getSequelizeModelAttributes(At),
+    AT_MODE_ATTRIBUTES: getSequelizeModelAttributes(AtMode),
+    AT_VERSION_ATTRIBUTES: getSequelizeModelAttributes(AtVersion),
+    BROWSER_ATTRIBUTES: getSequelizeModelAttributes(Browser),
+    BROWSER_VERSION_ATTRIBUTES: getSequelizeModelAttributes(BrowserVersion),
+    ROLE_ATTRIBUTES: getSequelizeModelAttributes(Role),
+    TEST_PLAN_ATTRIBUTES: getSequelizeModelAttributes(TestPlan),
+    TEST_PLAN_REPORT_ATTRIBUTES: getSequelizeModelAttributes(TestPlanReport),
+    TEST_PLAN_RUN_ATTRIBUTES: getSequelizeModelAttributes(TestPlanRun),
+    TEST_PLAN_TARGET_ATTRIBUTES: getSequelizeModelAttributes(TestPlanTarget),
+    TEST_RESULT_ATTRIBUTES: getSequelizeModelAttributes(TestResult),
+    USER_ATTRIBUTES: getSequelizeModelAttributes(User),
+    USER_ROLES_ATTRIBUTES: getSequelizeModelAttributes(UserRoles)
+};
```

Comment:
Based on what I said elsewhere about the attribute filtering not being needed, it might be possible to remove this file!